validator-ui/src/pages/homepage.jsx

```diff
@@ -32,7 +78,34 @@ export default function Homepage() {
           },
         ]}
       />
-      <div className="flex flex-col items-center justify-center h-screen"></div>
+      <div className="h-screen m-10">
+        <div className="flex justify-between flex-wrap">
+          <div>
+            <h1 className="text-4xl">Companii</h1>
+            <p className="font-semibold">600 de rezultate</p>
+          </div>
+          <div className="hidden md:block">
+            <label for="sort" className="mr-4">
```

Comment:
`for` nu este recunoscut ca "property" 
```suggestion
            <label htmlFor="sort" className="mr-4">
```